src/process.c

```diff
@@ -113,6 +113,115 @@ char * rcutils_get_executable_name(rcutils_allocator_t allocator)
   return executable_name;
 }
 
+#if defined _WIN32 || defined __CYGWIN__
+static
+rcutils_ret_t
+append_backslashes(rcutils_char_array_t * char_array, size_t n)
```

Comment:
at 1st glance i could not imagine what this size for, how about the following?

```suggestion
append_backslashes(rcutils_char_array_t * char_array, size_t num_backslashes)
```

Reply:
Sure, done in 2238b12.